deploy/kubernetes/base/controller.yaml

```diff
@@ -62,6 +62,23 @@ spec:
             timeoutSeconds: 3
             periodSeconds: 10
             failureThreshold: 5
+        - name: node-update-controller
+          image: quay.io/powercloud/ibm-powervs-block-csi-driver:v0.0.1
+          command: ["/node-update-controller"]
+          ports:
+            - name: metrics
+              containerPort: 8081
+              protocol: TCP
+            - name: healthz
+              containerPort: 8082
+              protocol: TCP
```

Comment:
@Prajyot-Parab are we exposing metrics and healthz from the controller?

Reply:
yes [here](https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/blob/a6a21787de3b139e131651f152968c8cdab80416/adhoc-controllers/main.go#L53-L54)

Reply:
cool, then @Madhan-SWE can we add `livenessProbe` probe then.?